services-geojson/src/main/java/com/mapbox/geojson/GeoJson.java

```diff
@@ -7,7 +7,7 @@
  *
  * @since 1.0.0
  */
-public interface GeoJson {
+public abstract class GeoJson {
```

Comment:
geojson itself is public and changing it from an interface to an abstract class would be a breaking change as users using it will need to change `implements` to `extends` + if they are already extending an object, they won't be able to make this change due to java only supporting single inheritance.